src/main/java/com/sourcery/gymapp/backend/workout/repository/RoutineRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.sourcery.gymapp.backend.workout.repository;
+
+import com.sourcery.gymapp.backend.workout.model.Routine;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface RoutineRepository extends JpaRepository<Routine, UUID> {
+
+    Routine getRoutineById(UUID routineId);
+    List<Routine> getRoutinesByUserId(UUID userId);
```

Comment:
same thinking as in controller. Should we be able to download somebodys routines?
It suppose to be download all my routines.
getAllRoutinesForUser or sth like this

Reply:
Can't change this one because the naming is mapped by JPA